docker/pg_tests/Dockerfile_config_test

```diff
@@ -0,0 +1,22 @@
+
+FROM wal-g/golang:latest as build
+
+WORKDIR /go/src/github.com/wal-g/wal-g
+
+RUN apt-get update &&     apt-get install --yes --no-install-recommends --no-install-suggests     liblzo2-dev
+
+COPY vendor/ vendor/
+COPY internal/ internal/
+COPY cmd/ cmd/
+COPY main/ main/
+COPY utility/ utility/
+
+RUN cd main/pg &&     go build -race -o wal-g -tags lzo -ldflags "-s -w -X main.BuildDate=`date -u +%Y.%m.%d_%H:%M:%S`"
+
+FROM wal-g/pg:latest
+
+COPY --from=build /go/src/github.com/wal-g/wal-g/main/pg/wal-g /usr/bin
+
+COPY docker/pg_tests/scripts/ /tmp
+
+CMD su postgres -c "/tmp/tests/config_test.sh"
```

Comment:
All these files are different only in the last line. So it would be better to place a common part to Dockerfile_test. And place "FROM ..." in all Dockerfile_some_test files